kubernetes/config/incluster_config.py

```diff
@@ -16,13 +16,15 @@
 
 from kubernetes.client import configuration
 
-_SERVICE_HOST_ENV_NAME = "KUBERNETES_SERVICE_HOST"
-_SERVICE_PORT_ENV_NAME = "KUBERNETES_SERVICE_PORT"
-_SERVICE_TOKEN_FILENAME = "/var/run/secrets/kubernetes.io/serviceaccount/token"
-_SERVICE_CERT_FILENAME = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+from .config_exception import ConfigException
 
+SERVICE_HOST_ENV_NAME = "KUBERNETES_SERVICE_HOST"
+SERVICE_PORT_ENV_NAME = "KUBERNETES_SERVICE_PORT"
+SERVICE_TOKEN_FILENAME = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+SERVICE_CERT_FILENAME = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
 
-def _join_host_port(host, port):
+
+def join_host_port(host, port):
```

Comment:
I'm assuming you removed the ``_`` prefix because it is now imported by the test module.  That isn't necessary - a leading underscore indicating 'private' is a good convention but one that unit tests do not have to respect.  Unless you want to have to support this function for users outside this module, I suggest reverting the name change.